### PR TITLE
PIM-8222: Fix product model issues when SKU contains / (create variant through UI and get product models via API)

### PR DESCRIPTION
**Pull request by pierallard**
See

https://github.com/akeneo/pim-community-dev/pull/8258/commits/58f5fe0091955d26589f570e30281470572647ba

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | y
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: CHANGELOG-2.3.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # 2.3.x
 
+## Bug fixes
+
+- PIM-8222: Fix product model issues when SKU contains `/` (create variant through UI and get product models via API)
```
Comment (Doodoune) [inline CHANGELOG-2.3.md]: It's not SKU, but "code" for product model :)

Comment (jmleroux) [reply CHANGELOG-2.3.md]: GTM after this fix

**Comment by pierallard** [comment]
@jmleroux See description, there is a link to another PR where the same question was answered :)

File: CHANGELOG-2.3.md (hunk 1 above)
Comment (pierallard) [inline CHANGELOG-2.3.md]:
```suggestion
- PIM-8222: Fix product model issues when code contains `/` (create variant through UI and get product models via API)
```

**Pull request merged**